### ISSUE TITLE
how to disable the thing which shows while mining

### ISSUE THREAD

**Issue by eXpertdUst19** [cabaletta/baritone#2820]
how to disable the thing which show the ores are and which block it will gousingwile mining

# set rendergoal false does not work

**Comment by ZacSharp** [comment]
`#renderGoal false` turns off the green boxes
`#renderPath false` turns off the red line
Don't open another duplicate, you won't get a better answer unless you give us something to work with (a video of it not working would be nice)

**Comment by ZacSharp** [comment]
Duplicate of #2818
